app/main.py

```diff
@@ -1,3 +1,98 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int | float) -> None:
+        self.km = km
```

Comment:
What if user passes wrong data type to our class? Should we have some validation in place?